src/app/screens/Library/index.tsx

```diff
@@ -1,16 +1,40 @@
-import React from 'react';
 import BookItem from './components/BookItem';
+import React from 'react';
+import {FlatList, View} from 'react-native';
+import {LibraryData} from '../../../constants/constants';
+import LibrarySeparator from './components/LibrarySeparator';
+import listStyles from './styles';
 
+interface dataItem {
+  item: {
+    id: number;
+    author: string;
+    title: string;
+    genre: string;
+    publisher: string;
+    year: string;
+    image_url: string | null;
+  };
+}
 const Library = () => {
-  return (
+  const renderItem = ({item}: dataItem) => (
     <BookItem
-      title="A Little Bird Told Me"
-      author="Timothy Cross"
-      uri={{
-        uri: 'http://wolox-training.s3.amazonaws.com/uploads/6942334-M.jpg',
-      }}
+      key={item.id}
+      title={item.title}
+      author={item.author}
+      uri={item.image_url}
     />
   );
+  return (
+    <View style={listStyles.container}>
+      <FlatList
+        data={LibraryData}
+        renderItem={renderItem}
+        keyExtractor={(item: {id: any}) => item.id}
```

Comment:
extract this function as a constant arrow function like renderItem

Reply:
Done!